config/thresholds/mtr-2026.yml

```diff
@@ -78,3 +78,4 @@ fixed_employment_allowance: 66.0
 employment_income_variance: 60.0
 subject_matter_of_dispute_disregard: 999_999_999_999
 pension_contribution_cap: 5
+housing_benefit_gross: true
```

Comment:
@starswan Can we come up with a name that reflect boolean(ness). Eg: `housing_benefit_is_gross` or `is_housing_benefit_gross`